pypi_formula_mappings.json

```diff
@@ -244,6 +244,9 @@
     "package_name": "huggingface-hub",
     "exclude_packages": ["PyYAML", "typing-extensions"]
   },
+  "instalooter": {
```

Comment:
this should be instaloader 😄 

fixed in #155483

Reply:
`instaloader` was added [4 days ago](https://github.com/Homebrew/homebrew-core/pull/154329), so this is correct.